src/.gitignore

```diff
@@ -8,4 +8,4 @@ dist/
 *.egg-info/
 *.bak
 static/jsi18n/
-
+pretix/plugins/sendmail/migrations/**
```

Comment:
As commented, we don't need this line (it actually would be harmful, as the database tables wouldn't ever  be created for our users). However, as we don't need the model any more anyways...